index.html

```diff
@@ -1 +1,3 @@
-<h1>first add</h1>
\ No newline at end of file
+<h1>first add</h1>
+<h2>I don't know wut should I say</h2>
```

Comment:
hey @ArhabAlShargabi why in earth did you add this . please refactor code and submit again.